lib/mayaUsd/fileio/jobs/jobArgs.cpp

```diff
@@ -723,21 +737,14 @@ UsdMayaJobExportArgs UsdMayaJobExportArgs::CreateFromDictionary(
 
 /* static */
 MStatus UsdMayaJobExportArgs::GetDictionaryFromEncodedOptions(
-    const MString&       optionsString,
-    VtDictionary*        toFill,
-    std::vector<double>* timeSamples)
+    const MString& optionsString,
+    VtDictionary*  toFill)
 {
     if (!toFill)
         return MS::kFailure;
 
     VtDictionary& userArgs = *toFill;
 
-    bool       exportAnimation = false;
```

Comment:
The animation frame are now extracted in a separate function since the functionality needs to be used from multiple places.